Marsiple/Comment/CommentView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CommentView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 23/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class CommentView: UIView {
+    private let maxCharacterCount = 200
+    private let textView = UITextView.autolayoutView()
+    private let keyboardSizedView = UIView.autolayoutView()
+    private let characterCountLabel = UILabel.autolayoutView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillChangeFrame),
+                                               name: NSNotification.Name.UIKeyboardWillChangeFrame,
+                                               object: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+}
+
+private extension CommentView {
+    @objc func keyboardWillChangeFrame(notification: NSNotification) {
+        if let userInfo = notification.userInfo {
+            let endFrame = (userInfo[UIKeyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue
```

Comment:
Since the `keyboardFrame` is `essential` for our logic, I'd say we should `guard` on both the `userInfo` and `endFrame`.